mailpoet/assets/css/src/components-automation-editor/_step.scss

```diff
@@ -33,3 +33,19 @@
 .mailpoet-automation-editor-step-subtitle {
   color: inherit;
 }
+
+.colored-icon {
```

Comment:
I think we should add the `mailpoet-automation` prefix (we seem to do it in other places across the MailPoet codebase).

Reply:
Addressed in [b455854](https://github.com/mailpoet/mailpoet/pull/4275/commits/b455854cda6b723906a9272563811b23dc0b2f4b)